.travis.yml

```diff
@@ -44,14 +40,17 @@ jobs:
   include:
     - stage: Lint-and-Coverage
       script: ../.travis/lint
+    - # parallel
       script: ../.travis/coverage
-    - stage: Firefox-Unit-Tests-Local
+    - stage: Local-Browser-Tests
       script: ../.travis/test Firefox
-    - stage: Chrome-Unit-Tests
+    - # parallel
+      script: ../.travis/test Chrome_travis_ci      
```

Comment:
The reason why (it was me) the tests were run in stages one after the other was that the tests sometime fail with timeouts caused by slow running containers at travis or saucelab. If the first test in firefox succeeded and the chrome test failed afterwards it was sufficent to restrat the chrome test via the travis web interface until it succeeded. If you are running firefox and chrome in parallel the probability that one of the test fails and you have to rerun the stage with both tests again is higher. The drawback is that the test will take a longer time if the tests run sequentially in stages.

Reply:
I also suspect that the travis test have cpu quota which potentially lead to container run interruptions or lower cpu priority if you run tests in parallel, at least if you are a non paying customer.

Reply:
you can restart the individual jobs within a stage